src/stylus/fablevision/fv-gamete.styl

```diff
@@ -8,10 +8,13 @@
   align-items: center
 
 .sperm .gamete-container
-  background: url('../resources/fablevision/layout-c/sperm_head.png')
-  background-size: contain
+  background-image: url('../resources/fablevision/layout-c/sperm_head.png'),
+                    url('../resources/fablevision/layout-c/sperm_tail.png')
+  // pin the tail on the sperm
```

Comment:
It actually took a non-trivial amount of time to line the images up properly. I thought about switching to the single whole-sperm image, but it has a white background (not transparent) so it wouldn't have worked without getting new art from FableVision.